managers/oclcClassify.py

```diff
@@ -67,10 +67,14 @@ def getClassifyResponse(self):
 
         return self.parseXMLResponse()
 
+    # TODO: can we query by everything?
```

Comment:
I'm not sure. We could try it, but it's good to have these query methods separated so it would make it easier to error handle.